### ISSUE TITLE
The name snek

### ISSUE THREAD

**Issue by BlobMoss** [snarkusius/lua#1]
Why would you name pong snek?

**Comment by Flipsterr** [comment]
Yeah that's weird why would you do that? I have some suggestions:
Pongathon (the goal is to get to 42.2 thousand in score)
Pongsters
MLP (Major League Pong)
